src/components/NavigationBar/index.js

```diff
@@ -198,6 +198,7 @@ class NavigationBar extends Component {
       searchSelectWidth: this.getSelectMenuWidth(this.props.searchType),
       showSearchBar: !isMobileView(1000),
       rightContainer: true,
+      isMobile400: isMobileView(400),
```

Comment:
No need to store it in state variable. Just declare a variable named `mobileView` inside render

Reply:
done.